.github/workflows/ci-windows.yml

```diff
@@ -7,7 +7,36 @@ permissions:
 
 jobs:
   Windows:
-    name: Windows
+    name: Windows (Native OPAM)
+    runs-on: windows-latest
+    steps:
+      - name: 🔭 Checkout code
+        uses: actions/checkout@v4
+
+      - name: 🐫 Setup OCaml
+        uses: ocaml/setup-ocaml@v3
+        with:
+          ocaml-compiler: 4.14
+          dune-cache: true
+          opam-local-packages: |
+            coqide-server.opam
+            rocq-*.opam
+
+      - name: 🐫🐪🐫 Get Rocq dependencies
+        run: opam install --deps-only ./coqide-server.opam ./rocq-*.opam
+
+      - name: 🧱 Build Rocq
+        run: opam exec -- make world
+
+      - name: 🐫 Get test suite deps
+        run: opam install ounit2 conf-python-3 conf-time
+
+      - name: 🐛 Test Rocq
+        run: opam exec -- make -j 4 -C test-suite DISABLED_SUBSYSTEMS='misc coq-makefile precomputed-time-tests'
+
+  Windows-platform:
```

Comment:
Should we remove this?

Reply:
Removed, I think we did agree to do this in the call, but excuse me if my memory fails.

Reply:
Not sure we were explicit about it but anyway we can easily put it back if we ever want to